packages/lyve_cloud/data_stream/audit/_dev/test/pipeline/test-audit-events.json-expected.json

```
@@ -0,0 +1,659 @@
+{
+    "expected": [
+        {
+            "@timestamp": "2022-10-24T08:37:41.232Z",
+            "cloud": {
+                "provider": "Seagate's Lyve Cloud"
```

```suggestion
                "provider": "lyvecloud"
```

This field is a keyword and would be better for searching to stick with the suggested format https://www.elastic.co/guide/en/ecs/current/ecs-cloud.html#field-cloud-provider

---

changed